packages/perseus-editor/src/widgets/interactive-graph-editor/components/interactive-graph-settings.tsx

```diff
@@ -75,11 +75,12 @@ type Props = {
 
     /**
      * The type of markings to display on the graph.
+     * - axes: shows the axes without the gride lines
      * - graph: shows the axes and the grid lines
      * - grid: shows only the grid lines
      * - none: shows no markings
      */
```

Comment:
Since the possible values are not actually defined on this line anymore, it seems like this comment might drift out of sync with the code. How would you feel about documenting the different values where `MarkingsType` is defined (in perseus-types.ts)?